SmartDeviceLink/private/SDLTextAndGraphicManager.m

```diff
@@ -195,12 +191,16 @@ - (void)sdl_updateAndCancelPreviousOperations:(BOOL)supersedePreviousOperations
     [self.transactionQueue addOperation:updateOperation];
 }
 
-- (void)sdl_updatePendingOperationsWithNewScreenData:(SDLTextAndGraphicState *)newScreenData {
+- (void)sdl_updatePendingOperationsWithNewScreenData:(SDLTextAndGraphicState *)newScreenData errorState:(SDLTextAndGraphicState *_Nullable)errorData {
```

Comment:
I _think_ that this can be separated into two different methods. It appears that the operation will only pass either an error or new current screen data. So this could be two different methods:

`sdl_updatePendingOperationsWithNewScreenState:`
and
`sdl_updatePendingOperationsWithFailedScreenState:`